LibRangeCheck-3.0/LibRangeCheck-3.0.lua

```diff
@@ -339,6 +339,7 @@ else
   tinsert(HarmSpells.WARLOCK, 18223) -- Curse of Exhaustion (Affliction) (30/33/36/35/38/42 yards)
   tinsert(HarmSpells.WARLOCK, 689) -- Drain Life (Affliction) (20/22/24 yards, level 14, rank 1)
   tinsert(HarmSpells.WARLOCK, 403677) -- Master Channeler (Affliction) (20/22/24 yards, level 14, rank 1)
+  tinsert(HarmSpells.WARLOCK, 426320) -- Shadowflame (30/33/36/39/42 yards, level 14, rank 1)
```

Comment:
This spell doesn't exist on cata classic. I think this needs an additional check

Reply:
I was under the impression that this is okay. There are a few other warlock spells in the list which don't exist in all game versions. 403677 is similarly only usable in Season of Discovery on Era. Does it behave differently than when a spell exists but isn't known?